apps/server/src/modules/rocketchat-user/service/rocket-chat-user.service.ts

```diff
@@ -14,6 +17,11 @@ export class RocketChatUserService {
 	}
 
 	public deleteByUserId(userId: EntityId): Promise<number> {
-		return this.rocketChatUserRepo.deleteByUserId(userId);
+		this.logger.log({ action: 'Deleting rocketChatUser ', userId });
+		const deletedRocketChatUser: Promise<number> = this.rocketChatUserRepo.deleteByUserId(userId);
+
+		this.logger.log({ action: 'Deleted rocketChatUser ', userId });
```

Comment:
Please log some more metadata (as for the _classes_ cases where I've left some example).

Reply:
OK I can only now add info about number of deleted userId. More deteiled information about updated documents will be added in ticket https://ticketsystem.dbildungscloud.de/browse/BC-6114